src/order/filter/basic.lean

```diff
@@ -1470,6 +1470,10 @@ lemma eventually_le.le_sup_of_le_right [semilattice_sup β] {l : filter α} {f g
   h ≤ᶠ[l] f ⊔ g :=
 by filter_upwards [hg] with x hgx using le_sup_of_le_right hgx
 
+lemma eventually_le.add_le_add {α β : Type*} [ordered_semiring β] {l : filter α}
```

Comment:
Why do you need an ordered semiring? Would the hypotheses of `add_le_add` suffice?

Also, could you place this lemma next to `eventually_le.mul_le_mul`?

Reply:
I've copied whats done for `mul_le_mul` and obtained `add_le_add` through `to_additive` instead